files/image_config/get_graph/get_graph

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+while [ ! -f /etc/sonic/graph_service_url ]; do
+    echo "No graph service url speficied, waiting for DHCP response..."
+    sleep 1
+done
```

Comment:
where is the graph service configuration file to enable/disable graph downloading?